cmake/shared/PACE_CodeAnalysis.cmake

```diff
@@ -2,10 +2,14 @@ add_custom_target(analyse
   COMMENT "Performing code analysis..."
   )
 
+# Handle CMAKE list to matlab cell array
+string(JOIN "','" IGNORE_STRING ${PACE_MLINT_IGNORE})
```

Comment:
You're now dependent on this variable being set; that should be noted in the function header. There are good examples of this in `PACE_Version.cmake`

Reply:
If unset, it will be a blank string, which is exactly what we want.